nixos/modules/services/mail/schleuder.nix

```diff
@@ -0,0 +1,100 @@
+{ config, pkgs, lib, ... }:
+let
+  cfg = config.services.schleuder;
+  settingsFormat = pkgs.formats.yaml { };
+  postfixMap = entries: lib.concatStringsSep "\n" (lib.mapAttrsToList (name: value: "${name} ${value}") entries);
+  writePostfixMap = name: entries: pkgs.writeText name (postfixMap entries);
+in
+{
+  options.services.schleuder = {
+    enable = lib.mkEnableOption "Schleuder secure remailer";
+    lists = lib.mkOption {
+      description = "Addresses of lists that should be handled by Schleuder.";
+      type = lib.types.listOf lib.types.str;
+      default = [ ];
+      example = [ "widget-team@example.com" "security@example.com" ];
+    };
+    /* maybe one day....
+      domains = lib.mkOption {
+      description = "Domains for which mail should be handled by Schleuder.";
+      type = lib.types.listOf lib.types.str;
+      default = [];
+      example = ["securelists.example.com"];
+      };
+    */
```

Comment:
Not sure if this would be sane, but could be parsed from the list of lists.

Also where would this need to be configured? Looks to me like schleuder doesn't really care about domains.

Reply:
See https://schleuder.org/schleuder/docs/server-admins.html#configuration -- dedicating an entire domain to schleuder is suggested. This isn't appropriate for the use case currently tested, where users and secure lists coexist on the same domain.

Reply:
Made the description of the unimplemented option clearer :)